.github/workflows/tagging.yml

```
@@ -19,17 +19,17 @@ jobs:
           echo $commitmsg
       - name: get tag version
         run: |
-          echo ::set-env name=tagversion::$( sed -e 's#.*: \(\)#\1#' <<< ${commitmsg} )
-          echo $tagversion
+          echo ::set-env name=tagversion::$( egrep -o '([0-9]+\.){2}(\*|[0-9]+)(-.*)?' <<< ${commitmsg} )
```

don't include white space characters
```suggestion
          echo ::set-env name=tagversion::$( egrep -o '([0-9]+\.){2}(\*|[0-9]+)(-\S*)?' <<< ${commitmsg} )
```